_posts/en/newsletters/2023-12-20-newsletter.md

```diff
@@ -0,0 +1,986 @@
+[apr taproot channels]: /en/newsletters/2023/08/30/#lnd-7904
+[apr watchtower]: /en/newsletters/2023/04/05/#watchtower-accountability-proofs
+[aug antidos]: /en/newsletters/2023/08/09/#denial-of-service-dos-protection-design-philosophy
+[aug bdk]: /en/newsletters/2023/08/09/#bdk-0-28-1
+[aug btcpay]: /en/newsletters/2023/08/02/#btcpay-server-1-11-1
+[aug cln]: /en/newsletters/2023/08/30/#core-lightning-23-08
+[aug combo]: /en/newsletters/2023/08/30/#covenant-mashup-using-txhash-and-csfs
+[aug fraud]: /en/newsletters/2023/08/23/#fraud-proofs-for-outdated-backup-state
+[aug fundingdos]: /en/newsletters/2023/08/30/#disclosure-of-past-ln-vulnerability-related-to-fake-funding
+[aug htlcendo]: /en/newsletters/2023/08/09/#htlc-endorsement-testing-and-data-collection
+[aug lnd taproot]: /en/newsletters/2023/08/30/#lnd-7904
+[aug milksad]: /en/newsletters/2023/08/09/#libbitcoin-bitcoin-explorer-security-disclosure
+[aug onion]: /en/newsletters/2023/08/09/#bolts-759
+[aug opreturn]: /en/newsletters/2023/08/09/#proposed-changes-to-bitcoin-core-default-relay-policy
+[aug payjoin]: /en/newsletters/2023/08/16/#serverless-payjoin
+[chatbtc]: https://chat.bitcoinsearch.xyz/
+[dec bcc]: /en/newsletters/2023/12/06/#bitcoin-core-26-0
+[dec cluster]: /en/newsletters/2023/12/06/#cluster-mempool-discussion
+[dec liqad]: /en/newsletters/2023/12/13/#discussion-about-griefing-liquidity-ads
+[dec payjoin]: /en/newsletters/2023/12/13/#payjoin-client-for-bitcoin-core-released
+[dec warnet announce]: /en/newsletters/2023/12/13/#bitcoin-network-simulation-tool-warnet-announced
+[dec zipkin warnet]: /en/newsletters/2023/12/06/#testing-with-warnet
+[feb bitcoinsearch]: /en/newsletters/2023/02/15/#bitcoinsearch-xyz
+[feb cln offers]: /en/newsletters/2023/02/08/#core-lightning-5892
+[feb codex32]: /en/newsletters/2023/02/22/#proposed-bip-for-codex32-seed-encoding-scheme
+[feb eclair offers]: /en/newsletters/2023/02/22/#eclair-2479
+[feb htlcendo]: /en/newsletters/2023/02/22/#feedback-requested-on-ln-good-neighbor-scoring
+[feb lnflag]: /en/newsletters/2023/02/22/#ln-quality-of-service-flag
+[feb miniscript]: /en/newsletters/2023/02/22/#bitcoin-core-24149
+[feb op_vault2]: /en/newsletters/2023/03/08/#alternative-design-for-op-vault
+[feb op_vault]: /en/newsletters/2023/02/22/#draft-bip-for-op-vault
+[feb payjoin]: /en/newsletters/2023/02/01/#serverless-payjoin-proposal
+[feb storage]: /en/newsletters/2023/02/15/#core-lightning-5361
+[jamming paper]: /en/newsletters/2022/11/16/#paper-about-channel-jamming-attacks
+[jan bip329]: /en/newsletters/2023/01/25/#bips-1383
+[jan eclair]: /en/newsletters/2023/01/04/#eclair-0-8-0
+[jan hwi]: /en/newsletters/2023/01/18/#hwi-2-2-0
+[jan inquisition]: /en/newsletters/2023/01/04/#bitcoin-inquisition
+[jan ord1]: /en/newsletters/2023/02/08/#discussion-about-storing-data-in-the-block-chain
+[jan ord2]: /en/newsletters/2023/02/15/#continued-discussion-about-block-chain-data-storage
```

Comment:
I just noticed that these appear in February newsletters. The first topic links to an email from January, but the second links to an email from February.